src/main/fc/fc_tasks.c

```diff
@@ -208,6 +209,16 @@ void taskVtxControl(uint32_t currentTime)
 }
 #endif
 
+#ifdef CAMERA_CONTROL
+void taskCameraControl(uint32_t currentTime)
+{
+    if (ARMING_FLAG(ARMED))
```

Comment:
Not sure if the check is even needed. After all, `cameraControlProcess` is quick to return (as it is now).